gems/smithy/spec/interfaces/client/waiters_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require_relative '../../spec_helper'
+
+describe 'Client: Waiters' do
+  let(:input) { { string_property: 'input_string' } }
+  let(:client) { Wait_Service::Client.new(stub_responses: true) }
+  let(:unexpected_error) { Smithy::Client::Waiters::Errors::UnexpectedError }
```

Comment:
Nit: unless you're using these in more than one place, it's more readable to stick into the test itself.